cmd/cosmos-sdk-cli/cmd/init.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/gobuffalo/packr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(initCmd)
+}
+
+func resolveProjectPath(remoteProjectPath string) string {
+	gopath := os.Getenv("GOPATH")
+	if gopath == "" {
+		gopath = build.Default.GOPATH
+		// Use $HOME/go
+	}
+	return gopath + string(os.PathSeparator) + "src" + string(os.PathSeparator) + remoteProjectPath
+}
+
+var initCmd = &cobra.Command{
+	Use:   "init AwesomeProjectName",
+	Short: "Initialize your new cosmos zone",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("Project name is required")
+		}
```

Comment:
Remove this check and instead use `Args:  cobra.ExactArgs(1),`
https://github.com/cosmos/cosmos-sdk/blob/90ed178c2bbcf9a42af96480243229c8ee1692c2/x/stake/client/cli/query.go#L22

Reply:
Thanks for sharing this. Will change.